### ISSUE TITLE
Add continuous integration (travis)

### ISSUE THREAD

**Issue by jgvictores** [roboticslab-uc3m/tools#4]
Add continuous integration (e.g. via travis).

**Comment by jgvictores** [comment]
Interesting link error using `static const int` instead of `define`: https://travis-ci.org/roboticslab-uc3m/tools/jobs/222320928

**Comment by jgvictores** [comment]
Done at 20270098741a07c806f0b10308da1f689485ee20.

**Comment by PeterBowman** [comment]
> Interesting link error using static const int instead of define: https://travis-ci.org/roboticslab-uc3m/tools/jobs/222320928

You were missing the actual definition of the static const members:

```c++
// PlaybackThread.cpp
const int teo::PlaybackThread::NOT_PLAYING = 0;
const int teo::PlaybackThread::PLAYING = 1;
```

Compare in-class initialization vs. definition concepts at http://stackoverflow.com/a/3026054.

**Comment by jgvictores** [comment]
@PeterBowman Yes, that was it! Implemented at abf2b035f407abf211707d83078fd40fb08c7111. My original (failed) intention was something in the lines of [this](https://github.com/asrob-uc3m/robotDevastation/blob/3a7b908a8cae43d3794f1482af5286d6ed6d4c9a/src/libraries/RobotLib/RobotManager.hpp#L59), but I guess that works because it's only used inside the header. Thanks once again!
